src/PresentationalComponents/Loading/Loading.spec.ct.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
When I open cypress, I dont see this test available. Only the rulesTable. Is this intentional because of how basic the test is? If so we should just remove it

Reply:
Forgot to change "spec.ct.js" to "cy.js"